eureka-core/src/main/java/com/netflix/eureka/DefaultEurekaServerContext.java

```diff
@@ -59,16 +59,20 @@ public DefaultEurekaServerContext(EurekaServerConfig serverConfig,
 
     @PostConstruct
     @Override
-    public void initialize() throws Exception {
+    public void initialize() {
         logger.info("Initializing ...");
         peerEurekaNodes.start();
-        registry.init(peerEurekaNodes);
+        try {
+            registry.init(peerEurekaNodes);
+        } catch (Exception e) {
+            logger.error("Error occurred while initializing registry", e);
```

Comment:
Wouldn't it be better to wrap and throw as a RuntimeException instead of suppressing with a log message?